backend/app/groups/routes.py

```
@@ -145,3 +155,64 @@ async def remove_group_member(
     if not removed:
         raise HTTPException(status_code=400, detail="Failed to remove member")
     return RemoveMemberResponse(success=True, message="Member removed successfully")
+
+
+@router.post("/{group_id}/image", response_model=ImageUploadResponse)
+async def upload_group_image(
+    group_id: str,
+    file: UploadFile = File(...),
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)